Utilities/StorageFactory/interface/StorageAccount.h

```diff
@@ -36,6 +38,7 @@ class StorageAccount {
     writeViaCache,
     writev
```

Comment:
If you add a `nOperations` item at the end, you can use that value in the `allOperations` array declaration. That solves the problem you were trying to avoid in your comments.